### DISCUSSION TITLE
Please tell me there is a way to remove freaking gray space from your monitor then you hide overlay from the screen man)))

### DISCUSSION THREAD

**Discussion by Esgareth** [baffler/Transparent-Twitch-Chat-Overlay#236] [General]
I update to latest version and regret dearly, i just want old thing back, cant even scroll to last stable version, this one is just absolute ****** (i dont have literature words for that)

Gray space on your monitor, what a nice idea))) And in settings you cant disable it))

**Comment by Esgareth** [comment]
Also you can't disable old version notification - YOU NEED TO UPDATE ASAP from old version as far as i understand as well..

**Comment by baffler** [comment]
Can you show a screenshot of the gray space? I'm not sure what you mean...

**Reply by Esgareth** [reply]
its for everybody - then you tap f7 you instead of having borders having gray space on your screen check it yourself

**Reply by laineylegend** [reply]
I have this exact same issue. There is a residue from the new app that was not there before - where it used to be transparent, it is now grey and hovers over the space. It also has the "x" marked, making it ugly and standout poorly. I upgraded to Windows 11, perhaps this caused the issue.

**Reply by laineylegend** [reply]
https://imgur.com/a/aIXDqs9 image of the issue

**Reply by laineylegend** [reply]
i actually apparently have it hovering over the same area twice getting it to work, but you get the idea. it is no longer transparent and makes the app unpleasant to use. also thank you for looking at this issue : )

**Comment by RazCallahan** [comment]
Its BARELY visible, and dont bother me that much, but would be nice to have option to turn it off completely😌

**Reply by RazCallahan** [reply]
I have increased contrast of screenshot a little for clearer understanding of what are we talking about.
![изображение](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/assets/30924974/780a4f5e-d457-42fc-806b-0f1e281cb977)

**Reply by RazCallahan** [reply]
It appears to be visible better on slightly darker backgrounds and completely disappears on lighter ones

**Comment by RazCallahan** [comment]
You fixed it! What a goat! THX!

**Reply by baffler** [reply]
> You fixed it! What a goat! THX!

Thanks! Btw, I found a couple issues with widgets (for one, the "x" issue was still there for widgets). So if you updated quickly and are having issues with widgets, you might have to uninstall and download the installer for v1.0.4 again to fix it.
